src/coroutine.rs

```diff
@@ -315,7 +344,9 @@ impl Drop for Handle {
         if state == State::Finished {
             ctx.resume(0);
         } else {
-            ctx.resume_ontop(self.0 as *mut _ as usize, coroutine_unwind);
+            self.unwind_flag.store(true, ::std::sync::atomic::Ordering::Release);
+            //ctx.resume_ontop(self.0 as *mut _ as usize, coroutine_unwind);
+            ctx.resume(self.0 as *mut _ as usize);
```

Comment:
Yes, you can pass some specific value to the `data` field to indicate that you are going to do force unwinding. The `data` will be returned from the `resume` in `yield_with`.